packages/core/src/redux/slices/roomConnection.ts

```diff
@@ -234,20 +234,23 @@ export const selectShouldConnectRoom = createSelector(
         selectRoomConnectionStatus,
         selectSignalConnectionDeviceIdentified,
         selectLocalMediaStatus,
+        selectRoomConnectionError,
     ],
     (
         appIsActive,
         hasOrganizationIdFetched,
         roomConnectionStatus,
         signalConnectionDeviceIdentified,
         localMediaStatus,
+        roomConnectionError,
     ) => {
         if (
             appIsActive &&
             localMediaStatus === "started" &&
             signalConnectionDeviceIdentified &&
             !!hasOrganizationIdFetched &&
-            ["ready", "reconnecting", "disconnected"].includes(roomConnectionStatus)
+            ["ready", "reconnecting", "disconnected"].includes(roomConnectionStatus) &&
+            !roomConnectionError
```

Comment:
I don't know if there's a scenario where there's an error but we still want to retry to join the meeting. 🤔 If there is, please let me know and I make this condition more strict by adding

```js
roomConnectionError !== "room_full"
```

Reply:
maybe host presence controls? I don't think it's a big issue though - the SDK should be mostly used by embedded accounts where that is disabled anyways.